data/856/331/29/85633129.geojson

```diff
@@ -22,6 +22,9 @@
     "itu:country_code":[
         "34"
     ],
+    "label:eng_x_preferred_placetype":[
+        "country"
+    ],
```

Comment:
Not wrong, but just curious where this was sourced from since its the country.. existing placetype backfill?

Reply:
- For WOF `country` I manually set them to `country` (with 11 exceptions for "disputed territory", "crown dependency", "special administrative area")
- For WOF `dependency` I manually set them to researched values either by reviewing Natural Earth, Statoids, Wikipedia, or government documents. Values range from "territory" to "overseas territory" to "territorial collectivity" to "external territory" to "commonwealth" and a few others)